test/unit/full_text_search/searcher_test.rb

```diff
@@ -82,5 +82,41 @@ def test_syntax_error_query
       assert_equal([issue],
                    targets.collect(&:source_record).sort_by(&:id))
     end
+
+    def test_results_ordered_by_descending_registered_time
```

Comment:
Can we simplify this name?

```suggestion
    def test_order_registered_time_desc
```

Reply:
fix: [8fcf70f](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/8fcf70f2f2109880e50cd3c84d872326c8f448ff)